rb-files/app.rb

```diff
@@ -23,72 +19,63 @@ def list_books
     end
   end
 
-  def show_person(person)
-    print "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
-    puts ", Specialization: #{person.specialization}" if person.is_a?(Teacher)
-    puts ", Classroom: #{person.classroom}" if person.instance_of?(Student)
-  end
-
   def list_people
     return puts 'No people found' if @people.empty?
 
     @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
   end
 
   def create_person
-    print 'Do you want to Create a students (1) or a teacher (2)? [Input the number]: '
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
     select_person = gets.chomp.to_i
     print 'Name: '
     name = gets.chomp
     print 'Age: '
     age = gets.chomp.to_i
```

Comment:
- [ ] The previous collector refers to this part of the code, All requests for input and input validation in your methods should've been placed in the main.rb file and/or classes/methods that are responsible for requesting or validating input. 
which means that this part of the code should be moved to a class or method which specifically handles user input.